_pages/en_US/troubleshooting.md

```diff
@@ -80,3 +80,76 @@ If this doesn't fix it, or there is no update downloaded, you may need to downlo
 
 If this doesn't work, ask on the r/Jailbreak [Discord Server](https://discord.gg/jb) for support.
 {: .notice--info}
+
+## <a name="sileo_errors" /> Fixing package installation errors on Sileo
+
+For the purposes of this troubleshooting section, we are assuming you are running Sileo on Taurine/Odyssey/Odysseyra1n.
+
+In order to determine which error you are encountering, you will have to tap `Show Details` in Sileo when you encounter an error with installing or removing a package.
+
+If the fixes below don't help, feel free to ask on the [Sileo Discord](https://discord.gg/Sileo).
```

Comment:
Just saying, for the sake of formatting and consistency, wouldn't it be better to word this as:
> If the fixes below don't help, feel free to ask on **Sileo [Discord Server](https://discord.gg/Sileo) for support**

Reply:
I've changed that sentence so it fits the format better 👍